default.nix

```diff
@@ -26,6 +16,4 @@ in rec {
   nixos = import ./nixos;
 
   path = ./.;
-
-  inherit (flake) inputs;
```

Comment:
@ncfavier Was the `inputs` here intended as a field for public consumption. I'm a bit weary about removing it now that it has been published but I don't see what purpose it has as a public field so perhaps it is not used much?

Reply:
No it was internal.

Reply:
Sweet. Thanks!